lib/private/AppFramework/Bootstrap/RegistrationContext.php

```diff
@@ -300,6 +312,10 @@ public function registerTwoFactorProvider(string $appId, string $class): void {
 		$this->twoFactorProviders[] = new ServiceRegistration($appId, $class);
 	}
 
+	public function registerCalendarProviders(string $appId, string $class): void {
```

Comment:
```suggestion
	public function registerCalendarProvider(string $appId, string $class): void {
```

because apps register *one*